guides/v2.2/migration/migration-troubleshooting.md

```diff
@@ -128,17 +128,15 @@ Run the Data Migration Tool in `Delta` mode to transfer incremental changes.
 ### Deltalog is not installed
 
 {%highlight xml%}
-Deltalog for <DATABASE> is not installed
+Deltalog for <TABLE_NAME> is not installed
 {%endhighlight%}
 
 #### Explanation
 
-This error occurs during the process of incremental migration of changes to data. It means the delta migration of changes made in Magento 1 since the last time you migrated data was not performed properly and retained.
+This error occurs during the process of [incremental migration]({{ page.baseurl }}/migration/migration-migrate-delta.md) of changes to data. It means deltalog tables (with prefix `m2_cl_*`) were not found in Magento 1 database. The tool installs these tables during [data migration]({{ page.baseurl }}/migration/migration-migrate-data.md) as well as database triggers which track changes and fill deltalog tables.
 
-Delta migrations can fail when you wipe the Magento 1 database before doing the delta, since this action erases the tables created during the initial migration of data into the Magento 1 database. These tables are needed to perform the delta migration properly. 
+One reason for the error could be that you are trying to migrate from a *copy* of your live Magento 1 store, not from live store itself. When you make a copy of the M1 live store DB, that copy does not contain the triggers and additional deltalog tables in the DB, and the migration fails. It fails because the delta migration mode does NOT make comparisons between the DB of M1 and M2 and then migrate the differences. Instead, the DMT uses triggers and additional deltalog tables to perform the migration. And these triggers are not within the copy because you may not have run a migration yet on the live DB from which you made the copy. In such a case, your copy will not contain the triggers and deltalog tables that the DMT needs to perform a migration.
```

Comment:
One reason for the error could be that you are trying to migrate from a *copy* of your live Magento 1 store, not from **the** live store itself.